lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -48,48 +48,67 @@ using namespace PlatformAgnostic::ICUHelpers;
 
 #endif // INTL_ICU
 
-// NOTE(jahorto): Keep these enums in sync with those by the same name in Intl.js
-// These enums are used by both WinGlob- and ICU-backed Intl
-enum class NumberFormatStyle
-{
-    Decimal, // Intl.NumberFormat(locale, { style: "decimal" }); // aka in our code as "number"
```

Comment:
Is it worth preserving these comments about the meaning/usage of the value?

Reply:
_Meh._. I don't think they were particularly descriptive before.

Reply:
I can add them back if others feel differently.